File: Software Technologies/Exam Practice - 02 Sept 2017/CSharp/ShoppingList/Models/Product.cs
using System.ComponentModel.DataAnnotations;

namespace ShoppingList.Models
{
    public class Product
    {
        public int Id { get; set; }

        [Required]
        public int Priority { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public int Quantity { get; set; }

        [Required]
        public string Status { get; set; }
    }
}

File: Software Technologies/Exam Practice - 02 Sept 2017/JS/models/Product.js
const mongoose = require('mongoose');

let productSchema = mongoose.Schema({
    priority: {type: 'number', required: 'true'},
    name: {type: 'string', required: 'true'},
    quantity: {type: 'number', required: 'true'},
    status: {type: 'string', required: 'true'}
});

let Product = mongoose.model('Product', productSchema);

module.exports = Product;